pyproject.toml

```diff
@@ -1,7 +1,19 @@
 [build-system]
-requires = ['setuptools', 'setuptools_scm', 'wheel']
+requires = ['setuptools', 'setuptools_scm', 'wheel', 'pytoml']
 build-backend = 'setuptools.build_meta'
 
+# we’ll move to flit, but for now this only holds package data
+[tool.flit.metadata]
+author = 'Philipp Angerer, Alex Wolf, Isaac Virshup, Sergei Rybakov'
+# We don’t need all emails, the main authors are sufficient.
+author-email = 'phil.angerer@gmail.com, f.alex.wolf@gmx.de'
```

Comment:
Can this be stored in a more tool independent way? Like how for `setup.cfg` there's the `metadata ` and `options.extras_require` tables.